pkg/pfcpsim/session/urr_ies.go

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2024-present Ian Chen <ychen.cs10@nycu.edu.tw>
+package session
+
+import (
+	"encoding/binary"
+
+	"github.com/wmnsk/go-pfcp/ie"
+)
+
+//  Measurement Information IE bits definition
```

Comment:
The following change will fix the lint issue
```suggestion
// Measurement Information IE bits definition
```